tests/ttnn/unit_tests/operations/ccl/test_all_reduce_t3000_frequent.py

```diff
@@ -18,6 +19,8 @@ def is_unsupported_case(input_shape, math_op, mem_config, num_devices, num_links
     num_l1_banks = 64
     if mem_config.buffer_type == ttnn.BufferType.L1 and tensor_size_bytes > num_l1_banks * 50 * 1024:
         return True, "L1 buffer can't support large tensor sizes"
+    if (input_shape[2] == 32 or input_shape[3] == 32) and input_dtype == ttnn.bfloat8_b:
+        return True, "This combination is not supported for now"
```

Comment:
I recommend to not use normalized dims and use -1, -2 instead. Our tensors our ND rank in general now.

Reply:
Also please use `TILE_HEIGHT`, `TILE_WIDTH` instead of hardcoded values